src/namespaces/apoc/cypher/run-first-column.ts

```diff
@@ -92,12 +92,15 @@ class RunFirstColumnFunction extends CypherFunction {
     }
 
     private convertArrayToParams(env: CypherEnvironment, variables: Variable[]): string {
-        const params = variables.reduce((acc, variable) => {
-            const globalScopeName = variable.getCypher(env);
-            const key = env.getReferenceId(variable);
-            acc[key] = globalScopeName;
-            return acc;
-        }, {} as Record<string, string>);
+        const params = variables.reduce(
+            (acc, variable) => {
+                const globalScopeName = variable.getCypher(env);
+                const key = env.getReferenceId(variable);
+                acc[key] = globalScopeName;
+                return acc;
+            },
+            {} as Record<string, string>
+        );
```

Comment:
```suggestion
        const params = variables.reduce(
            (acc: Record<string, string>, variable) => {
                const globalScopeName = variable.getCypher(env);
                const key = env.getReferenceId(variable);
                acc[key] = globalScopeName;
                return acc;
            },
            {}
        );
```

I know it's not the main point of this PR but I just wanted to point ou the "more correct" way of assigning a type in a reducer.